NuKeeper.Integration.Tests/NuGet/Api/BulkPackageLookupTests.cs

```diff
@@ -48,6 +48,25 @@ public class BulkPackageLookupTests
             Assert.That(results, Does.ContainKey("Newtonsoft.Json"));
         }
 
+        [Test]
+        public async Task FindsSingleUpdateForPackageDifferingOnlyByCase()
+        {
+            var packages = new List<PackageIdentity>
+            {
+                Current("nunit"),
+                Current("NUnit")
+            };
+
+            var lookup = BuildBulkPackageLookup();
+
+            var results = await lookup.FindVersionUpdates(
+                packages, NuGetSources.GlobalFeed, VersionChange.Major);
+
+            Assert.That(results, Is.Not.Null);
+            Assert.That(results.Count, Is.EqualTo(1));
+            Assert.That(results, Does.ContainKey("NUnit"));
+        }
```

Comment:
Can you also ` Assert.That(results, Does.ContainKey("nunit"));` if that's true?

Reply:
`Count` is equal to `1`, only the canonical package name from NuGet is returned. Oh, but you mean lookup now allows searching by both? Yeah, that makes sense.

Reply:
Oh. This assert ignores the dictionary comparer and uses the default one. That's naughty `NUnit`!

Reply:
Yes, if the key matching is case-insensitive then the dictionary should reply "yes, I have this key" for either caps.

Reply:
Would ` Assert.That(results.ContainsKey("nunit", Is.True)` work?